evm-contracts/src/v0.6/dev/BlockhashStore.sol

```diff
@@ -0,0 +1,67 @@
+pragma solidity 0.6.2;
+
+/**
+ * @title BlockhashStore
+ * @notice This contract provides a way to access blockhashes older than
+ * the 256 block limit imposed by the BLOCKHASH opcode.
+ * You may assume that any blockhash stored by the contract is correct.
+ */
+contract BlockhashStore {
+
+  mapping(uint => bytes32) internal s_blockhashes;
+
+  /**
+   * @notice stores blockhash of a given block, assuming it is available through BLOCKHASH
+   * @param n the number of the block whose blockhash should be stored
+   */
+  function store(uint256 n) public {
```

Comment:
[Seems like we're no longer consistent about it](https://github.com/smartcontractkit/chainlink/blob/develop/evm-contracts/src/v0.6/Chainlink.sol#L50), but I thought we were prefixing method argument names with underscores, like `_n`.

Reply:
The solidity style guide changed its mind about this. `_prefixedNames` are now out of vogue